cmds/record.c

```diff
@@ -1497,6 +1494,52 @@ static void print_child_usage(struct rusage *ru)
 # define EM_AARCH64  183
 #endif
 
+static bool is_regular_executable(const char *pathname)
+{
+	struct stat sb;
+
+	if (!stat(pathname, &sb))
+		if (S_ISREG(sb.st_mode) && (sb.st_mode & S_IXUSR))
+			return true;
```

Comment:
I prefer adding curly brackets to the outer if statement when body is more than one line (even when not strictly needed).

Reply:
Okay.